### PR TITLE
Adding GHOST_DATABASE_SSL variable to enable SSL configuration in Ghost

### PR DESCRIPTION
**Pull request by ordenador**
**Description of the change**

Enables SSL database connection.

**Benefits**

SSL connection configuration support to mysql database, covering the 3 example cases of the official documentation (boolean, string, json): https://ghost.org/docs/config/#ssl-1

**Possible drawbacks**

Requires MySQL client to be SSL enabled or  configured (MYSQL_CLIENT_ENABLE_SSL=true)

**Applicable issues**

**Additional information**

Tested in Azure database with SSL connection

### CODE REVIEW

**Comment by ordenador** [comment]
@danoo @carlossm @migmartri @megawebmaster can you review the PR? plz

File: 4/debian-10/rootfs/opt/bitnami/scripts/ghost-env.sh [hunk 1]
```diff
@@ -125,5 +125,9 @@ GHOST_DATABASE_USER="${GHOST_DATABASE_USER:-"${MARIADB_DATABASE_USER:-}"}"
 export GHOST_DATABASE_USER="${GHOST_DATABASE_USER:-bn_ghost}" # only used during the first initialization
 GHOST_DATABASE_PASSWORD="${GHOST_DATABASE_PASSWORD:-"${MARIADB_DATABASE_PASSWORD:-}"}"
 export GHOST_DATABASE_PASSWORD="${GHOST_DATABASE_PASSWORD:-}" # only used during the first initialization
+GHOST_DATABASE_SSL="${GHOST_DATABASE_SSL:-false}"
+echo $GHOST_DATABASE_SSL | jq -e . > /dev/null 2>&1 && rc=$? || rc=$? # check variable if no valid to jq
+if [ $rc -eq 4 ]; then GHOST_DATABASE_SSL=$(echo $GHOST_DATABASE_SSL | jq -eR .); fi # transform to json format if is string
```
[the remark this answers is not in the document]
Comment (ordenador) [reply 4/debian-10/rootfs/opt/bitnami/scripts/ghost-env.sh:130]: yes, you are right. It´s better to separate in 2 variable
I will make the changes and place the validation in `ghost_validate`
thanks @alemorcuq

**Comment by ordenador** [comment]
thank you for your comment @megawebmaster , you are right about the dual configuration.
I have just deleted the GHOST_DATABASE_SSL_CA variable to avoid confusions

**Comment by juan131** [comment]
@ordenador did you see my latest comments?

**Pull request closed without merging**